cli/popper/gha.py

```diff
@@ -537,18 +543,27 @@ def __init__(self, action, workspace, env, dry):
         self.cwd = os.getcwd()
 
     def run(self, reuse=False):
-        cmd = self.action.get('runs', ['entrypoint.sh'])
-        cmd[0] = os.path.join('./', cmd[0])
-        cmd.extend(self.action.get('args', []))
-
         cwd = self.cwd
-        if not self.dry_run:
-            if 'repo_dir' in self.action:
-                os.chdir(self.action['repo_dir'])
-                cmd[0] = os.path.join(self.action['repo_dir'], cmd[0])
-            else:
-                os.chdir(os.path.join(cwd, self.action['uses']))
-                cmd[0] = os.path.join(cwd, self.action['uses'], cmd[0])
+        if self.action['uses'] == 'sh':
+            cmd = self.action.get('runs')
+            cmd.extend(self.action.get('args', []))
+
+            if not self.dry_run:
+                os.chdir(cwd)
+                if cmd[0].startswith('./'):
```

Comment:
in the case of `uses = "sh"`, this does not apply, since we are running from the root of the project

Reply:
If we need to execute a bash script located in the root, then how to reference it ?

Reply:
do you mean something like the following?

```hcl
action "run on host" {
  uses = "sh"
  runs = "./scriptatroot"
}
```

Reply:
can also be:

action "run on host" {
  uses = "sh"
  runs = "scriptatroot"
}

Reply:
@ivotron Thanks! Got it.

Reply:
@ivotron So can we set the default runs to `sh -c`  i.e when `runs` attribute not provided ?

Reply:
yeah, that's a good idea!